app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,21 @@
+class PostsController < ApplicationController
+  
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = current_user.posts.build(content: params[:post][:content])
+    if @post.save 
+      redirect_to root_url
+    else
+      render 'post'
+    end
+  end
+
+  def index
+  end
```

Comment:
- [ ] All the posts should be  listed somewhere, being the index action the one returning them, [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/posts2.png) is the requirement

Reply:
Kindly fix that, please